beacon-chain/rpc/beacon_server.go

```diff
@@ -152,7 +153,7 @@ func (bs *BeaconServer) Eth1Data(ctx context.Context, _ *ptypes.Empty) (*pb.Eth1
 		//   (iii) newer than state.latest_eth1_data.block_data.
 		// vote.eth1_data.deposit_root is the deposit root of the eth1.0 deposit contract
 		// at the block defined by vote.eth1_data.block_hash.
-		isBehindFollowDistance := blockHeight.Add(blockHeight, big.NewInt(eth1FollowDistance)).Cmp(currentHeight) >= -1
+		isBehindFollowDistance := big.NewInt(0).Sub(currentHeight, big.NewInt(eth1FollowDistance)).Cmp(blockHeight) >= 0
```

Comment:
This is the correct way to do big int subtraction - the prior was giving us errors and weird behavior